README.md

```diff
@@ -108,160 +111,178 @@ See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthroug
+the VMware virtual machine. Also, as the onscreen instructions describe,
+the last step in configuration allows editing the resulting configuration file.
+
+|Setting|Value|Notes|
+|---|---|---|
+|*Instructions*|↵||
+|Company Name|Clavius|*Can substitute with your choice.*|
+|Region of Datacenter|orbit|*Can substitute with your choice.*|
+|Name of Datacenter|coal-1|(Availability zone.) *Can substitute with your choice.* |
+|Location of DataCenter|Moon, Earth|*Can substitute with your choice.*|
+|*Instructions*|↵||
+|'admin' interface|2|The second NIC is set up as the admin network by the CoaL networking script|
+|(admin) headnode IP address|10.99.99.7|Must use this value.|
+|(admin) headnode netmask:|↵|Use the default.|
+|(admin) Zone's starting IP address:|↵|Use the default.|
+|Add external network now? (Y/n)|Y|Must use this value.|
+|'external' interface|1|The first NIC is set up as the external network by the CoaL networking script|
+|(external) headnode IP address|10.88.88.200|Must use this value.|
+|(external) headnode netmask:|↵|Use the default.|
+|(external) gateway IP address:|10.88.88.2|Must use this value.|
+|(external) network VLAN ID|↵|Use default. The external network is not on a VLAN in CoaL|
+|Starting Provisionable IP address for external Network|↵|Use the default.|
+|Ending Provisionable IP address for external Network|↵|Use the default.|
+|Default gateway IP address:|↵|Use the default.|
+|Primary DNS Server|↵|Use the default.|
+|Secondary DNS Server|↵|Use the default.|
+|Head node domain name|example.com|*Can substitute with your choice.*|
+|DNS Search Domain|example.com|*Can substitute with your choice.*|
+|NTP Server IP Address|↵|Use the default.|
+|"root" password|rootpass|*Can substitute with your choice.*|
+|Confirm "root" password|||
+|"admin" password|adminpass1|*Can substitute with your choice.*|
+|Confirm "admin" password|||
+|Administrator's email|↵|Use the default.|
+|Support email|↵|Use the default.|
+|Confirm password|||
+|Enable telemetry|"true" or "false"|*Can use your choice*|
+|Verify Configuration|||
+|Verify Configuration Again|||
+
+- CoaL will now install based on the configuration parameters entered
+  above. Installation has been observed to take up to 20 minutes,
+  particularly if slow laptop HDD.
+
+After setup is complete you should be able to SSH into your CoaL on the
+"admin" network. Example:
 
-    ```bash
-    ssh root@10.99.99.7  # password 'root'
-    ```
+```bash
+ssh root@10.99.99.7  # password 'rootpass'
+```
 
 For just a taste run `svcs` to see running [SMF
 services](http://wiki.smartos.org/display/DOC/Using+the+Service+Management+Facility).
 Run `vmadm list` to see a list of current VMs (SmartOS
 [zones](http://wiki.smartos.org/display/DOC/Zones)). Each SDC service runs in
-its own zone. See [the SDC operator guide](https://docs.joyent.com/sdc7).
+its own zone. See [the Joyent customer operator guide](https://docs.joyent.com/sdc7).
```

Comment:
When we have open source operator documentation, I'll change it back.

I'm changing this to "Joyent customer operator guide" because there was one github ticket where the https://docs.joyent.com/sdc7 led someone to email support to get the PI. docs.jo/sdc7 hasn't yet been updated to continue the benefits-of-open-source tone to the joyent offers found on the www.jo site and blog. Search is confusing and so is the integration with https://www.joyent.com/developers.

It is confusing if you are coming at it from this side, although good documentation on the whole.